src/components/DragDrop/DragDropTypes.ts

```diff
@@ -21,4 +21,5 @@ export interface ContainerType {
   itemIds: string[];
   header?: React.ReactNode;
   emptyContent?: React.ReactNode;
+  columnClassName?: string;
```

Comment:
could we use string[] to allow for more than one to be added? (if so, the prop name might be changed to columnClassNames, since we use itemIds)

Reply:
Good point. Thanks for this @csnizik

Reply:
Made that change @csnizik . Thanks again!